docs/installation.rst

```diff
@@ -23,14 +23,16 @@ Setting up Vagrant
 
      >vagrant up
 
-  Note: If you encounter an error saying "It appears your machine doesn't support NFS, or there is not an
-  adapter to enable NFS on this machine for Vagrant.", then you need to install ``nfs-kernel-server`` using the command:
+  **Troubleshooting**: If you encounter an error saying *"It appears your machine doesn't support NFS, or there is not an adapter to enable NFS on this machine for Vagrant."* there could be two possible causes. You should first install ``nfs-kernel-server`` if it is not already installed, using the command:
 
   .. code-block:: bash
 
     apt-get install nfs-kernel-server
 
-* Go grab a tea or coffee, it will take a few minutes to setup the environment.
+  If the problem persists after a reboot and a ``vagrant destroy`` and ``vagrant up`` you may also need to reboot, enter your computers' Setup, and enable your BIOS's Virtualization Technology (vt-x), then invoke another ``vagrant up``.
```

Comment:
So yeah, this is a seperate issue. Basically we should say if you see the following text: 

  The guest machine entered an invalid state while waiting for it
  to boot. Valid states are 'starting, running'. The machine is in the
  'poweroff' state. Please verify everything is configured
  properly and try again.

... then try enabling the vt-x in the BIOS settings and try `vagrant up` again. 

There should be no need to run vagrant destroy for this error. Just re-running vagrant up should do the trick.